components/setup/RepoSelection.tsx

```diff
@@ -100,12 +100,12 @@ const RepoSelection = ({ repoProvider, installId, setIsRepoSelectionDone }: { re
 					setError('Something went wrong');
 				} else {
 					console.info(`[RepoSelection/handleSubmit] repos data saved successfully in db`);
-					setIsRepoSelectionDone(true);
+					if(setIsRepoSelectionDone) {setIsRepoSelectionDone(true) };
 				}
 			})
 			.catch((error) => {
 				setError(`Unable to submit selected repos, \nPlease refresh this page and try again.`);
-				setIsRepoSelectionDone(false);
+				if(setIsRepoSelectionDone) {setIsRepoSelectionDone(false) };
```

Comment:
can we check if the function exists before both the state setting?

Reply:
done

Reply:
Had to revert this. Can't return error if setIsRepoSelectionDone  is null